src/VS.Mefx/Commands/Command.cs

```diff
@@ -0,0 +1,77 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace VS.Mefx.Commands
+{
+    using System;
+    using System.Collections.Generic;
+    using Microsoft.VisualStudio.Composition;
+
+    /// <summary>
+    /// A general command class which serves a parent class for all the commands that can be run by application .
+    /// </summary>
+    internal class Command
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Command"/> class.
+        /// </summary>
+        /// <param name="derivedInfo">The ConfigCreator for the input files.</param>
+        /// <param name="arguments">The command line arguments from the user.</param>
+        public Command(ConfigCreator derivedInfo, CLIOptions arguments)
+        {
+            this.Creator = derivedInfo;
+            this.Options = arguments;
+        }
+
+        /// <summary>
+        /// Write the given lines in sorted order to the user.
+        /// </summary>
+        /// <param name="lines">The lines to output to the user.</param>
+        protected void WriteLines(List<string> lines)
+        {
+            lines.Sort();
```

Comment:
This has the side effect of reordering the collection the caller was holding, which per the doc comment on this method, isn't obvious.
Instead, you could copy the collection and sort that, or use the `OrderBy` extension method and then loop over that with `foreach`.